cmd/kube-scheduler/app/options/options.go

```diff
@@ -45,14 +45,21 @@ import (
 	kubeschedulerconfigv1alpha2 "k8s.io/kube-scheduler/config/v1alpha2"
 	schedulerappconfig "k8s.io/kubernetes/cmd/kube-scheduler/app/config"
 	"k8s.io/kubernetes/pkg/client/leaderelectionconfig"
-	"k8s.io/kubernetes/pkg/master/ports"
 	"k8s.io/kubernetes/pkg/scheduler"
 	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 	kubeschedulerscheme "k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config/validation"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/interpodaffinity"
 )
 
+const (
+	// KubeSchedulerPort is the default port for the scheduler status server.
+	// May be overridden by a flag at startup.
+	// Copied from pkg/master/ports/ports.go
+	// TODO: Import this constant from a consts only package, that does not pull any further dependencies.
+	KubeSchedulerPort = 10259
```

Comment:
Adding to https://github.com/kubernetes/kubernetes/pull/90000#pullrequestreview-390753146, this is the only location that `KubeSchedulerPort` is used, so it could be removed from pkg/master/ports. Though I think this may be better declared in one of the APIs than `cmd/kube-scheduler`, but I'll let someone else comment on that. @Huang-Wei wdyt?

Reply:
IMO we should move this along with `InsecureSchedulerPort` to `pkg/scheduler/apis/config/types.go`.